<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls</class>
 <widget class="QDialog" name="Controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string comment="Controls window title">Controls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <widget class="QLabel" name="PSP_Img">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>501</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/resources/psp.png</pixmap>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Up">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Left">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Right">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Down">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Home">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Start">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_Select">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>240</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_X">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_S">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_O">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_T">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>100</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_RT">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QKeyEdit" name="Edit_LT">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="PSP_Img_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>501</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/resources/psp_map.png</pixmap>
      </property>
     </widget>
     <zorder>PSP_Img</zorder>
     <zorder>PSP_Img_2</zorder>
     <zorder>Edit_Up</zorder>
     <zorder>Edit_Left</zorder>
     <zorder>Edit_Right</zorder>
     <zorder>Edit_Down</zorder>
     <zorder>Edit_Home</zorder>
     <zorder>Edit_Start</zorder>
     <zorder>Edit_Select</zorder>
     <zorder>Edit_X</zorder>
     <zorder>Edit_S</zorder>
     <zorder>Edit_O</zorder>
     <zorder>Edit_T</zorder>
     <zorder>Edit_RT</zorder>
     <zorder>Edit_LT</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QKeyEdit</class>
   <extends>QLineEdit</extends>
   <header>qkeyedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Controls</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Controls</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
